datastore/monitoring/models.py

```diff
@@ -0,0 +1,72 @@
+from datetime import datetime, date
+from dataclasses import dataclass
+from django.db import models
+from django.contrib.postgres.fields import ArrayField
+
+
+@dataclass
+class PublisherMetrics:
+    total_grants: int
+    total_gbp: float
+    total_funders: int
+    total_recipient_individuals: int
+    total_recipient_organisations: int
+
+
+class PublisherMetricsRecord(models.Model):
+    timestamp = models.DateTimeField()
```

Comment:
I'm just wondering if there is a need to be able to group datasets/snapshots in a way that doesn't rely on datetime, how difficult would it be to remove an erroneous one later on if two were generated at the same time?

Reply:
That is a good point, especially since snapshots timestamps are the `.datetime` of the last GetterRun, so if we created 2+ snapshots of the same GetterRun, they'd all have the same timestamp.

I guess there's the implicit Django `.id` autoincrement, do we want to make it explicit like GetterRuns?

Reply:
(in a meeting we discussed an additional model that groups snapshots together for convenience)